### DISCUSSION TITLE
DIY Power Meter?

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [doudar/SmartSpin2k#368] [General]
Totally not related to the SS2K other than its relating to bike power.  But has anyone here ever attempted to build their own power meter for other bikes - rather than spending a few hundred or more on pedals?  I found this - https://sensitivus.com/products/diy-bike-power-meter/ that I might give a go, need to do more reading on it to see where I can get parts if they list the details, etc.

**Comment by doudar** [comment]
I know there’s a GitHub project where someone built one. I’ll see if I can find and post it here.

Here's the one I found on GitHub:

https://github.com/stevejarvis/powermeter

The one you posted looks interesting as well. Keep us posted!
